.idea/.idea.Libplanet/.idea/aws.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="accountSettings">
+    <option name="activeProfile" value="profile:default" />
+    <option name="activeRegion" value="us-east-2" />
+    <option name="recentlyUsedProfiles">
+      <list>
+        <option value="profile:default" />
+      </list>
+    </option>
+    <option name="recentlyUsedRegions">
+      <list>
+        <option value="us-east-2" />
+      </list>
+    </option>
+  </component>
+</project>
```

Comment:
I believe this file was added by mistake.

Reply:
https://github.com/planetarium/libplanet/pull/1809/commits/f094c9a65b9402e2da546818e5eb076eb004b3ab amended